addons/udes_stock/models/stock_inventory.py

```diff
@@ -38,6 +45,14 @@ class StockInventory(models.Model):
         "Start Date Scanning",
         help="The date operator started scanning."
     )
+    user_id = fields.Many2one(
+        "res.users",
+        string="Checked Owner",
+        readonly=True,
+        states={"draft": [("readonly", False)], "waiting": [("readonly", False)]},
+        help="Specify Owner to focus your stock check on a particular user.",
```

Comment:
Super NIT: owner and not Owner

Reply:
This field is a copy paste from udes_stock_count and commented on udes_stock_count :).
Happy to make the change though :)

Reply:
Last chance: Do i need to change it as in the same way is used even in another help :)?

Reply:
Keep it consistent